AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -26,13 +28,16 @@ public static void configureAppCenter(Application application) {
         if (AppCenter.isConfigured()) {
             return;
         }
-        AppCenterReactNativeShared.sApplication = application;
-        AppCenterReactNativeShared.sWrapperSdk.setWrapperSdkVersion(com.microsoft.appcenter.reactnative.shared.BuildConfig.VERSION_NAME);
-        AppCenterReactNativeShared.sWrapperSdk.setWrapperSdkName(com.microsoft.appcenter.reactnative.shared.BuildConfig.SDK_NAME);
+        sApplication = application;
+        sWrapperSdk.setWrapperSdkVersion(com.microsoft.appcenter.reactnative.shared.BuildConfig.VERSION_NAME);
+        sWrapperSdk.setWrapperSdkName(com.microsoft.appcenter.reactnative.shared.BuildConfig.SDK_NAME);
         AppCenter.setWrapperSdk(sWrapperSdk);
+        if (!AppCenterReactNativeShared.isStartedAutomatically()) {
```

Comment:
As a code optimization, you can combine this line with line 28 and use a single return.

Reply:
No we need to set wrapper sdk and sApplication between the 2 checks actually.

Reply:
https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/383/files/7c92bd89bda8351d9e7e5146b1fddc28ebf1f0a7#r211803210